src/lib/types.ts

```
@@ -575,7 +576,7 @@ export interface DropdownProps extends PopperProps {
   isOpen?: boolean;
 }
 
-export interface DropdownDividerProps extends HTMLAttributes<HTMLDivElement> {}
+export interface DropdownDividerProps extends HTMLAttributes<HTMLDivElement> { }
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)